palamar/settings.py

```diff
@@ -195,6 +195,9 @@
 # file location
 FILE_STORAGE = os.path.join(BASE_DIR, "files")
 
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+STATIC_ROOT = os.path.join(PROJECT_DIR, 'static')
```

Comment:
In development branch (which is Master) we are not going to use "STATIC_ROOT" configuration variable. Instead we should keep using "STATICFILES_DIRS = [os.path.join(BASE_DIR, "static"),]".

In deployment or jenkins test phase, our Ansible script will remove STATICFILES_DIRS variable and add STATIC_ROOT variable.